letsencrypt/display/util.py

```diff
@@ -197,10 +215,12 @@ def menu(self, message, choices,
         :param choices: Menu lines, len must be > 0
         :type choices: list of tuples (tag, item) or
             list of descriptions (tags will be enumerated)
+        :param dict _kwargs: absorbs default / cli_args
+
+        :returns: tuple of (`code`, `index`) where
```

Comment:
Incorrect returns documentation. Compare it with the returns from the other `menu` functions.